metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -50,29 +50,17 @@ public MetacatEventBus(
         @Nonnull @NonNull final Registry registry
     ) {
         this.applicationEventMulticaster = applicationEventMulticaster;
-        this.eventAsyncCounter = registry.counter(Metrics.CounterEventAsync.getMetricName());
-        this.eventSyncCounter = registry.counter(Metrics.CounterEventSync.getMetricName());
-    }
-
-    /**
-     * Post event asynchronously.
-     *
-     * @param event event
-     */
-    public void postAsync(final ApplicationEvent event) {
-        log.debug("Received request to post an event {} asynchronously", event);
-        this.eventAsyncCounter.increment();
-        this.applicationEventMulticaster.postAsync(event);
+        this.eventPublishCounter = registry.counter(Metrics.CounterEventPublish.getMetricName());
     }
 
     /**
-     * Post event synchronously.
+     * Post event.
      *
      * @param event event
      */
-    public void postSync(final ApplicationEvent event) {
-        log.debug("Received request to post an event {} synchronously", event);
-        this.eventSyncCounter.increment();
-        this.applicationEventMulticaster.postSync(event);
+    public void post(final ApplicationEvent event) {
+        log.debug("Received request to post an event {}", event);
+        this.eventPublishCounter.increment();
```

Comment:
you combine these into one counter, what if the second async publish step failed. I guess assumption is that it won't fail.